### ISSUE TITLE
Nesting components breaks class preprocessing

### ISSUE THREAD

**Issue by steadicat** [ladderlife/om-css#30]
``` clojure
(dom/div (dom/div (dom/div {:class :component})))
```

becomes:

``` html
<div data-reactid=".d.0.1.0.0.0">
  <div data-reactid=".d.0.1.0.0.0.0">
    <div class=":component" data-reactid=".d.0.1.0.0.0.0.0">
    </div>
  </div>
</div>
```

**Comment by mikekap** [comment]
I think adding `nil`s for the params fixes it.

**Comment by steadicat** [comment]
@mikekap yup it does. That shouldn’t be necessary though because the first param is optional for `dom/*` and `defui` components. Making the props param optional was pretty much the whole point of `defcomponent`.

**Comment by anmonteiro** [comment]
fixed dd9e00e576c73e05c0e1bc20a7d9fea288c9d8b8
